tests/ui/pages/desktop/base.py

```diff
@@ -26,6 +26,15 @@ def login(self, email, password):
     def logout(self):
         self.header.click_logout()
 
+    def wait_for_page_to_load(self):
+        body = self.find_element(By.TAG_NAME, 'body')
+        self.wait.until(lambda s: 'firefox' in body.get_attribute('class'))
+
+    @property
+    def wait_for_page(self):
+        url = self.selenium.current_url
+        self.wait.until(lambda _: self.selenium.current_url != url)
```

Comment:
I tried using the example in PyPOM:
```
def wait_for_page_to_load(self):
                  body = self.find_element(By.TAG_NAME, 'body')
                  self.wait.until(lambda s: 'loaded' in body.get_attribute('class'))
```

This gets me a StaleElementException. How could I wait for something on the next page? It seems the page loads and then the url is the last thing to change. There doesn't seem to be anything injected onto the page either. I'll ask some of the devs